Gemfile

```diff
@@ -11,6 +11,11 @@ group :development, :test do
   gem 'pry'
 end
 
+group :lint, :test do
+  gem 'rubocop', '~> 0.82.0'
```

Comment:
I'd prefer allowing `rubocop` to update, as otherwise we'd leave it stuck on this version indefinitely

Reply:
You can connect a service like https://dependabot.com/ or https://depfu.com/ and you'll receive pull requests with updates. Then contributors with pull requests not related to RuboCop and Ruby code at all, as me, will not get failed CI.

Reply:
Sounds good 👍
Surprised we didn't do that sooner 😄
Opened #1150 for this